src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasRelationSlicer.java

```diff
@@ -907,6 +1022,8 @@ private void preProcessMultiPolygonRelation(final Relation relation)
         mergeOverlappingClosedMembers(relation, outers, inners);
     }
 
+    // TODO come back and verify we're keeping track of all required statistics
```

Comment:
Heh, we should at some point come back to this.. :)

Reply:
Sonar will do it for us!